### DISCUSSION TITLE
The equivalent rotation

### DISCUSSION THREAD

**Discussion by SMBEN** [mtex-toolbox/mtex#1427] [Ask Anything]
Hi;
When I worked with TSL OIM I did a rotation on the A1 axis (90°)to obtain the correct results. 
With Mtex I did a rotation by:
%define a rotation around x by 90 degrees
rot1 = rotation.byAxisAngle(xvector,90*degree)
Or: 
%define an axes based on a sample direction
v=vector3d(xvector);
%define a rotation
omega=90*degree;
%define your orientation
ori = orientation.byAxisAngle(v,omega,cs)
 
But the results are still incorrect
what can I do to solve this issue?
thank you.

**Comment by kilir** [comment]
Hi,
it remains a little unclear what you like to rotate, but in general, if you like to rotate something and if the result is not satisfying, most likely you need to find and apply a different rotation.
Cheers,
Rüdiger

**Comment by Tijmenvermeij** [comment]
It's also good to note that A1 is not always equal to the x-axis in MTex...

**Reply by SMBEN** [reply]
Hi, 
Thank you for your answers.
